pysatNASA/instruments/ace_sis_l2.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+"""Module for the Advanced Composition Explorer (ACE) SIS instrument.
+
+References
+----------
+- Stone, E., Frandsen, A., Mewaldt, R. et al. The Advanced Composition Explorer.
+  Space Science Reviews 86, 1–22 (1998). https://doi.org/10.1023/A:1005082526237
+- Stone, E., Cohen, C., Cook, W. et al. The Solar Isotope Spectrometer for the
+  Advanced Composition Explorer. Space Science Reviews 86, 357–408 (1998).
+  https://doi.org/10.1023/A:1005027929871
+
+Properties
+----------
+platform
+    'ace'
+name
+    'sis_l2'
+tag
+    None supported
+inst_id
+    ['h1', 'h2', 'k0']
+
+Note
+----
+- Level 1 ACE data is maintained at pysatSpaceWeather.
+
+Warnings
+--------
+- The cleaning parameters for the instrument are still under development.
+
+"""
+
+import datetime as dt
+import functools
+
+from pysat.instruments.methods import general as mm_gen
+
+from pysatNASA.instruments.methods import ace as mm_ace
+from pysatNASA.instruments.methods import cdaweb as cdw
+from pysatNASA.instruments.methods import general as mm_nasa
+
+# ----------------------------------------------------------------------------
+# Instrument attributes
+
+platform = 'ace'
+name = 'sis_l2'
+tags = {'': ''}
+inst_ids = {id: [''] for id in ['h1', 'h2', 'k0']}
```

Comment:
```suggestion
inst_ids = {id: [tag for tags.keys()] for id in ['h1', 'h2', 'k0']}
```

Recommend using the general implementation to reduce unforseen future changes.